e2e/tests/notebook/import-python-lib.spec.ts

```diff
@@ -0,0 +1,76 @@
+import {findWorkspace, signIn} from 'utils/test-utils';
+import DataPage, {TabLabelAlias} from 'app/page/data-page';
+import {makeRandomName} from 'utils/str-utils';
+import {config} from 'resources/workbench-config';
+import * as fs from 'fs';
+
+// Notebook server start may take a long time. Set maximum test running time to 20 minutes.
+jest.setTimeout(20 * 60 * 1000);
+
+describe('Import Python libraries', () => {
+
+  beforeEach(async () => {
+    await signIn(page);
+  });
+
+  test('Import os', async () => {
+    let codeSnippet;
+    try {
+      codeSnippet = fs.readFileSync('resources/python-code/import-os.txt', 'utf8');
+    } catch(e) {
+      throw new Error(e);
+    }
+
+    const workspaceCard = await findWorkspace(page);
+    await workspaceCard.clickWorkspaceName();
+
+    const dataPage = new DataPage(page);
+    const notebookName = makeRandomName('import-os');
+    const notebook = await dataPage.createNotebook(notebookName);
+    await notebook.waitForKernelIdle();
+
+    const cellIndex = 1;
+    const outputText = await notebook.runCodeCell(cellIndex, {code: codeSnippet});
+
+    // Delete notebook
+    const analysisPage = await notebook.goAnalysisPage();
+    await dataPage.openTab(TabLabelAlias.Analysis);
+    await analysisPage.deleteNotebook(notebookName);
+
+    // Partial values check: UserEmail and python3 path is expected to be printed out.
+    const partialArray = [
+      '/usr/local/bin/python3',
+      config.userEmail,
+    ];
+    const outputTextArray = outputText.split(/\n/);
+    expect(outputTextArray).toEqual(expect.arrayContaining(partialArray));
+  });
+
+  test('Import common lib', async () => {
+    let codeSnippet;
```

Comment:
Please move most of this to a helper. This is 95% identical to the other test method